build-tools/scripts/TestApks.targets

```diff
@@ -191,7 +195,7 @@
         Arguments="install-apks --modules _ALL_ --apks &quot;%(TestAab.Identity).apks&quot;"
         ContinueOnError="ErrorAndContinue"
         JavaPath="$(JavaPath)"
-        JarPath="$(BundleToolJarPath)"
+        JarPath="$(AndroidBundleToolJarPath)"
```

Comment:
Presumably this is the cause of the `error MSB4044`, and `$(AndroidBundleToolJarPath)` isn't set.